modules/quickstart/pages/quick_start.adoc

```diff
@@ -38,118 +36,181 @@ To use Stargate you need:
+Send a `POST` request to `/v2/schemas/keyspaces/{keyspace_name}/tables` to create a table.
+Set the table name and column definitions in the JSON body.
 
 [source,bash]
 ----
-curl --request POST \
-  --url 'localhost:8082/v2/schemas/keyspaces/betterbotz/tables/products/rows'
-  --header 'accept: application/json' \
-  --header 'content-type: application/json' \
-  --data '{"columns":[{"name":"id","value":"e9b6c02d-0604-4bab-a3ea-6a7984654631"},{"name":"name","value":"Heavy Lift Arms"},{"name":"description","value":"Heavy lift arms capable of lifting 1,250 lbs of weight per arm. Sold as a set."},{"name":"price","value":"4199.99"},{"name":"created","value":"2019-01-10 09:48:31.020+0040"}]}'
-----
-
-Send a `GET` request to `/v2/schemas/keyspaces/{keyspace_name}/tables/{table_name}/rows/{primary_key}` to retrieve a particular row from the table.
+curl --location --request POST 'localhost:8082/schemas/keyspaces/users_keyspace/tables' \
+--header 'X-Cassandra-Token: {auth-token-here}' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+	"name": "users",
+	"columnDefinitions":
+	  [
+        {
+	      "name": "firstname",
+	      "typeDefinition": "text"
+	    },
+        {
+	      "name": "lastname",
+	      "typeDefinition": "text"
+	    },
+	    {
+	      "name": "email",
+	      "typeDefinition": "text"
+	    },
+        {
+	      "name": "favorite color",
+	      "typeDefinition": "text"
+	    }
+	  ],
+	"primaryKey":
+	  {
+	    "partitionKey": ["firstname"],
+	    "clusteringKey": ["lastname"]
+	  },
+	"tableOptions":
+	  {
+	    "defaultTimeToLive": 0,
+	    "clusteringExpression":
+	      [{ "column": "lastname", "order": "ASC" }]
+	  }
+}'
+----
+
+=== Write data
+
+Send a `POST` request to `/v2/keyspaces/{keyspace_name}/{table_name}` to add data to the table.
+The column name / value pairs are passed in the JSON body.
 
 [source,bash]
 ----
-curl --request GET \
-  --url 'localhost:8082/v2/schemas/keyspaces/betterbotz/tables/products/rows/e9b6c02d-0604-4bab-a3ea-6a7984654631' \
-  --header 'accept: application/json'
+curl --location --request POST 'localhost:8082/v2/keyspaces/users_keyspace/users' \
+--header 'X-Cassandra-Token: {auth-token-here}' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "firstname": "Mookie",
+    "lastname": "Betts",
+    "email": "mookie.betts@gmail.com",
+    "favorite color": "blue"
+}'
 ----
 
-To retrieve all the rows in a table, send a `GET` request to `/v2/schemas/keyspaces/{keyspace_name}/tables/{table_name}/rows`.
+=== Read data
+
+Send a `GET` request to `/v2/keyspaces/{keyspace_name}/{table_name}` to retrieve a row using the primary key of the table.
 
 [source,bash]
 ----
-curl --request GET \
-  --url 'localhost:8082/v2/schemas/keyspaces/betterbotz/tables/products/rows' \
-  --header 'accept: application/json'
+curl -G --location --request GET 'http://localhost:8082/v2/keyspaces/users_keyspace/users' \
```

Comment:
Using `-G` and `--request GET` is redundant. In fact, with this request neither are probably needed.

Reply:
It looks like `-G` is needed

good
```
curl -G --location 'http://localhost:8082/v2/keyspaces/users_keyspace/users' \
> --header 'X-Cassandra-Token: {my auth token was here}' \
> --header 'Content-Type: application/json' \
> --data-urlencode 'where={"firstname": {"$eq": "Mookie"}}'
{"count":1,"data":[{"firstname":"Mookie","favorite color":"blue","email":"mookie.betts@gmail.com","lastname":"Betts"}]}
```

bad
```
curl --location 'http://localhost:8082/v2/keyspaces/users_keyspace/users' \
> --header 'X-Cassandra-Token: {my auth token was here}'' \
> --header 'Content-Type: application/json' \
> --data-urlencode 'where={"firstname": {"$eq": "Mookie"}}'
{"description":"Server error: Unrecognized token 'where': was expecting (JSON String, Number, Array, Object or token 'null', 'true' or 'false')\n at [Source: (String)\"where=%7B%22firstname%22%3A%20%7B%22%24eq%22%3A%20%22Mookie%22%7D%7D\"; line: 1, column: 6]","code":500}
```

bad
```
curl --location --request GET 'http://localhost:8082/v2/keyspaces/users_keyspace/users' \
> --header 'X-Cassandra-Token: {my auth token was here}' \
> --header 'Content-Type: application/json' \
> --data-urlencode 'where={"firstname": {"$eq": "Mookie"}}'
{"description":"Bad request: argument \"content\" is null","code":400}
```

Reply:
```
       -G, --get
              When used, this option will make all data specified with -d, --data, --data-binary or --data-urlencode to be used in an HTTP GET  request  instead  of  the
              POST request that otherwise would be used
```

Ah, that's why